monero/backends/jsonrpc.py

```diff
@@ -183,6 +184,47 @@ def balances(self, account=0):
         _balance = self.raw_request('getbalance', {'account_index': account})
         return (from_atomic(_balance['balance']), from_atomic(_balance['unlocked_balance']))
 
+    def get_incoming_transactions(
+        self,
+        account_index=0,
+        unconfirmed=False,
+        min_height=0,
+        subaddr_indices=None
+    ) -> List[IncomingPayment]:
+        """
+        Calls the wallet RPC to get incoming transactions.
+
+        Args:
+            account_index: The index of the account we want to query.
+            unconfirmed: Whether to include unconfirmed transactions (the ones
+            in the pool).
+            min_height: The height of the block where we want to start scanning
+            from.
```

Comment:
DocString is missing information about `subaddr_indices`.

Reply:
Good catch. It's updated now